src/pinv.jl

```diff
@@ -0,0 +1,102 @@
+# Moore-Penrose pseudoinverse
+
+"""
+    pinv(M; atol::Real=0, rtol::Real=atol>0 ? 0 : n*ϵ)
+    pinv(M, rtol::Real) = pinv(M; rtol=rtol) # to be deprecated in Julia 2.0
+
```

Comment:
I think the customary style in StaticArrays is not to add docstrings for things that simply overload a Base function (at least as long as the signatures are the same): i.e., I don't think we should have a doc-string for `pinv(::StaticMatrix)`

Reply:
Thanks for the reivew! I've removed the docstring.